src/Ionide.ProjInfo/Library.fs

```diff
@@ -750,22 +754,40 @@ module ProjectLoader =
         )
 
     let getNuGetReferences (p: ProjectInstance) =
-        p.Items
-        |> Seq.filter (fun p ->
-            p.ItemType = "Reference"
-            && p.GetMetadataValue "NuGetSourceType" = "Package"
-        )
-        |> Seq.map (fun p ->
-            let name = p.GetMetadataValue "NuGetPackageId"
-            let version = p.GetMetadataValue "NuGetPackageVersion"
-            let fullPath = p.GetMetadataValue "FullPath"
+        let references =
+            p.Items
+            |> Seq.filter (fun p ->
+                p.ItemType = "Reference"
+                && p.GetMetadataValue "NuGetSourceType" = "Package"
+            )
+            |> Seq.map (fun p ->
+                let name = p.GetMetadataValue "NuGetPackageId"
+                let version = p.GetMetadataValue "NuGetPackageVersion"
+                let fullPath = p.GetMetadataValue "FullPath"
 
-            {
-                Name = name
-                Version = version
-                FullPath = fullPath
-            }
-        )
+                {
+                    Name = name
+                    Version = version
+                    FullPath = fullPath
+                }
+            )
+
+        let packageReferences =
+            p.Items
+            |> Seq.filter (fun p -> p.ItemType = "PackageReference")
```

Comment:
Shout if you would like to commonise this. For example, I could `Seq.choose` into a `Choice1Of2` and `Choice2Of2` and then match on that to decide whether to use `EvaluatedInclude` or `NuGetPackageId`.